src/lbaf/Model/lbsObject.py

```diff
@@ -289,11 +295,29 @@ def get_unused_params(self) -> dict:
         """Return all current unused parameters."""
         return self.__unused_params
 
+    def __get_property_or_qoi_name(self, ftn_name) -> str:
+        """Return the QOI name from the given QOI getter function"""
+        name = ftn_name[4:] if ftn_name.startswith("get_") else ftn_name
+        if name == "size":
+            name = "object_memory"
+        elif name == "overhead":
+            name = "overhead_memory"
+        return name
+
+    def get_entity_properties(self) -> list:
+        """Get all methods decorated with the 'property' decorator.
+        """
+        property_methods : dict = {
+            self.__get_property_or_qoi_name(name): getattr(self, name)
+            for name in dir(self)
+            if callable(getattr(self, name)) and not name.startswith("__") and hasattr(getattr(self, name), "is_property") }
```

Comment:
Please add comment here to explain the logic of this test for future reference

Reply:
Fixed (improved docs for qoi getters)